algo/src/main/java/com/hartwig/serve/sources/iclusion/ActionableTrialFactory.java

```diff
@@ -19,6 +19,7 @@
 import org.apache.commons.compress.utils.Lists;
 import org.apache.logging.log4j.LogManager;
 import org.apache.logging.log4j.Logger;
+import org.apache.logging.log4j.util.Strings;
```

Comment:
I know this is used in a lot of places in our code, but this class is annotated `@InternalApi` with the description "Consider this class private": https://logging.apache.org/log4j/2.x/javadoc/log4j-api/org/apache/logging/log4j/util/Strings.html

I think it's better to just use `""` for the empty string instead of a potentially unstable dependency.